gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/GraviteeLicenseServiceImpl.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.am.management.service.impl;
+
+import static java.util.stream.Collectors.*;
+
+import io.gravitee.am.management.service.GraviteeLicenseService;
+import io.gravitee.node.api.Node;
+import io.gravitee.node.api.license.Feature;
+import io.gravitee.node.api.license.License;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+import io.reactivex.rxjava3.core.Single;
+import org.springframework.stereotype.Component;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * @author Antoine CORDIER (antoine.cordier at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+@Component
+public class GraviteeLicenseServiceImpl implements GraviteeLicenseService {
```

Comment:
This service is already provided by gravitee-node :)

Reply:
It's NodeLicenseService

Reply:
see https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/installation/GraviteeLicenseResource.java